src/Nfts/Nfts.stories.tsx

```diff
@@ -0,0 +1,145 @@
+import {storiesOf} from '@storybook/react-native'
+import BigNumber from 'bignumber.js'
+import React from 'react'
+import {QueryClient, QueryClientProvider} from 'react-query'
+import {Provider} from 'react-redux'
+
+import {mockWallet, poolInfoAndHistory, stakePoolId} from '../../storybook'
+import getConfiguredStore from '../legacy/configureStore'
+import {SelectedWalletProvider} from '../SelectedWallet'
+import {YoroiWallet} from '../yoroi-wallets'
+import {StakePoolInfosAndHistories} from '../yoroi-wallets/types'
+import {Nfts} from './Nfts'
+
+const mockedAccountState = {
+  isDelegating: false,
+  isFetching: false,
+  lastFetchingError: '',
+  poolOperator: null,
+  totalDelegated: new BigNumber(0),
+  value: new BigNumber(0),
+}
+
+storiesOf('Nfts', module)
+  .add('not delegating', () => {
```

Comment:
copypasta

Reply:
I will look into the storybook after, since I'll need a bit of help using it!